Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -28,9 +28,14 @@
   --remotion-themed-scrollbar-width: 14px;
   --remotion-themed-line-height: 1.2em;
   --remotion-themed-padding: 0.4em;
+  --remotion-themed-margin: 0.4em;
   --remotion-themed-min-height: calc(var(--remotion-themed-line-height) + 2 * var(--remotion-themed-padding));
   --remotion-themed-multiline-min-height: calc(2 * var(--remotion-themed-line-height) + 2 * var(--remotion-themed-padding));
   --remotion-themed-control-width: 16em;
+  --remotion-themed-blur-radius: 6px;
+  --remotion-themed-spread-radius: 2px;
+  --remotion-themed-box-shadow: 0 0 var(--remotion-themed-blur-radius) var(--remotion-themed-spread-radius) var(--color-contrast-separator);
```

Comment:
that's the popup-shadow. I think the styleguide uses 6px and 20px. please recheck and then do a padding-based variant for this, e.g. 4*padding etc. Please recheck in the styleguide

We also need to name those as remotion-themed-popup-shadow-blur-radius, etc